R/utilities-figures.R

```diff
@@ -343,26 +344,44 @@ createPlotsFromExcel <- function(simulatedScenarios, observedData, projectConfig
   })
   names(multiPanelPlots) <- dfPlotGrids$name
 
+  if (nrow(dfExportConfigurations) > 0) {
+    # create a list of ExportConfiguration objects from dfExportConfigurations
+    exportConfiguration <- createEsqlabsExportConfiguration(projectConfiguration)
```

Comment:
Please add a validation for export configurations, i.e., if all required fields are populated (output name) and the referenced plot grids are defined.